_includes/manuals/2.3/2_quickstart_guide.md

```diff
@@ -0,0 +1,20 @@
+
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup.  It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger(Optional since 2.1 version), TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
```

Comment:
here as well
```suggestion
Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger, TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
```